dbms/src/Interpreters/QueryNormalizer.cpp

```diff
@@ -91,14 +91,21 @@ void QueryNormalizer::visit(ASTIdentifier & node, ASTPtr & ast, Data & data)
             return;
         }
 
+        /// We are alias for other column (node.name), but we are alias by
```

Comment:
Cannot parse the comment. Are we an alias? Yes, we are. :)

Reply:
I also found this comment not clear enough... but still understandable.